ports/geckolib/build.rs

```diff
@@ -62,8 +62,7 @@ try:
     style_template.render()
 
     geckolib_template = Template(filename=os.environ['GECKOLIB_TEMPLATE'], input_encoding='utf8')
-    output = geckolib_template.render(STYLE_STRUCTS = style_template.module.STYLE_STRUCTS,
-                                      LONGHANDS = style_template.module.LONGHANDS)
```

Comment:
I looked around and couldn't find any actual usage of the `LONGHANDS` constant, so I dropped it.